test/init_api/init_api.cpp

```diff
@@ -25,7 +25,7 @@ void app_start(int argc, char *argv[])
 {
     (void) argc;
     (void) argv;
-    MBED_HOSTTEST_TIMEOUT(5);
+    MBED_HOSTTEST_TIMEOUT(60);
```

Comment:
This is the host test timeout configured as per greentea recommendations. 
- I only see minor benefit of making it configurable.